programs/lib/include/flexpret_exceptions.h

```diff
@@ -40,28 +40,32 @@ typedef void (*isr_t)(void);
  * @brief Enable interrupts for thread
  * 
  */
-void enable_interrupts();
+#define enable_interrupts() set_csr(CSR_STATUS, 0x10);
```

Comment:
Hmmm. Normally the convention in C is that every macro is upper case. This wasnt the case when you started working on the code-base. But I am thinking we might want to change to this? Also. My preference is that the macros shouldnt include semicolons. Also to make it safe to use in any context you should wrap it in do { } while(0): https://stackoverflow.com/questions/923822/whats-the-use-of-do-while0-when-we-define-a-macro